### PR TITLE
DIV-6839 Transfer to CTSC from Service journey / Bailiff states

### PR DESCRIPTION
**Pull request by ravinderpal**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-6839

Acceptance criteria
```
Transfer from CTSC to RDC
GIVEN a case is in any of these states: AwaitingBailiffReferral;AwaitingBailifService;IssueToBailiff;AwaitingAOSBailiff;AwaitingService Consideration,AwaitingServicePayment,AwaitingGeneralApplicationPayment,AwaitingGeneralConsideration,GeneralConsiderationComplete;DNPronounced
THEN a caseworker can trigger the event 'Transfer from CTSC to RDC'
AND a screen appears where the user can select a value from the drop-down for D8DivorceUnit (see attached screenshots)
AND when the event is submitted state is changed to 'Issued'
```

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json [hunk 1]
```diff
@@ -208,5 +208,16 @@
...
     "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "29/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "transferCaseFromCTSCService",
+    "Name": "Transfer from CTSC to RDC",
+    "Description": "Transfer from CTSC to an RDC",
+    "DisplayOrder": 11,
+    "PreConditionState(s)": "AwaitingBailiffReferral;AwaitingBailiffService;IssuedToBailiff;AwaitingServiceConsideration;AwaitingServicePayment;AwaitingGeneralReferralPayment;AwaitingGeneralConsideration;GeneralConsiderationComplete;DNPronounced",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json:219]: are you sure `caseworker-divorce-courtadmin_beta` role has CRU for all these states? (in fact just `RU` would be enough)?

File: definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json [hunk 2]
```diff
@@ -208,5 +208,16 @@
...
+    "PostConditionState": "Issued",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json:220]: are you sure `caseworker-divorce-courtadmin_beta` role has CRU for all this state? (in fact just `CR` would be enough)

**Review by yarpo — APPROVED** [review]

File: definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json (hunk 1 above)
Comment (ravinderpal) [reply definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json:219]: Yes, it does 👍

File: definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json (hunk 2 above)
Comment (ravinderpal) [reply definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json:220]: Yes, it does 👍

**Review by adamjhc — APPROVED** [review]

**Review by madjava — APPROVED** [review]

**Pull request merged**